part-spec/component.json

```diff
@@ -72,7 +72,7 @@
 				},
 				"thermal": {
 					"description": "component temperature and thermal resistance information",
-					"$ref": "./common/thermals.json#/thermalProperties"
+					"$ref": "./common/thermal.json#/thermal"
 				},
```

Comment:
I checked the part-spec/common folder and the changes in this component.json are aligned with the filename (thermal.json) and the object name( thermal) in the part-spec/common. The proposed changes will break the links. Is there a reason we want to revert?

Reply:
Yeah, I believe this was a problem but it's been fixed in https://github.com/edatasheets/digital-datasheets/pull/113/files. Take a look and let us know if it's still broken for you.